drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -147,13 +158,29 @@ def process(transaction):
 	sa = transaction.getSampleForUpdate(sampleID)
 	space = sa.getSpace()
 
-	# Get modality from metadata here
-	modality = "CT-BIOPSY"
-	exp = createNewImagingExperiment(transaction, space, project, modality)
-	# Get Omero id(s) from Omero before creating sample(s) in openBIS
+	metadataFile = None
+	for root, subFolders, files in os.walk(incomingPath):
+		for f in files:
+			stem, ext = os.path.splitext(f)
+			if ext.lower()=='.tsv':
+				with open(os.path.join(root, f), 'U') as fh: metadataFile = fh.readlines()
 	offset = 0
-	for f in omero_image:
-		omero_id = "omero-test-id"
-		createNewImagingRun(transaction, sa, exp, omero_id, offset)
+	for line in metadataFile[1:]:  # Exclude the header from iteration
```

Comment:
For me, it is not really clear from the code, what exactly happens here, nor what the metadataFile contains. Suggestion: Maybe write two or three comment lines, such that other developer know what happens here?